src/shared/schema/us/actions/favorites.ts

```diff
@@ -24,6 +26,11 @@ export const favoritesActions = {
         path: ({entryId}) => `${PATH_PREFIX}/favorites/${filterUrlFragment(entryId)}`,
         params: (_, headers) => ({headers}),
     }),
+    renameFavorite: createAction<RenameFavoriteResponse, RenameFavoriteArgs>({
+        method: 'POST',
+        path: ({entryId}) => `${PATH_PREFIX}/favorites/${filterUrlFragment(entryId)}/rename`,
```

Comment:
You don't need to use `filterUrlFragment` already, it's implemented in gateway

Reply:
Removed